docs/source/tutorials/e2e_flow.rst

```diff
@@ -75,14 +73,14 @@ For this tutorial, we'll fine-tune the model using LoRA. LoRA is a parameter eff
 technique which is especially helpful when you don't have a lot of GPU memory to play with. LoRA
 freezes the base LLM and adds a very small percentage of learnable parameters. This helps keep
 memory associated with gradients and optimizer state low. Using torchtune, you should be able to
-fine-tune a Llama2 7B model with LoRA in less than 16GB of GPU memory using bfloat16 on a
+fine-tune a Llama-3.2-3B-Instruct model with LoRA in less than 16GB of GPU memory using bfloat16 on a
 RTX 3090/4090. For more information on how to use LoRA, take a look at our
 :ref:`LoRA Tutorial <lora_finetune_label>`.
 
 We'll fine-tune using our
 `single device LoRA recipe <https://github.com/pytorch/torchtune/blob/main/recipes/lora_finetune_single_device.py>`_
 and use the standard settings from the
-`default config <https://github.com/pytorch/torchtune/blob/main/recipes/configs/llama2/7B_lora_single_device.yaml>`_.
+`default config <https://github.com/pytorch/torchtune/blob/main/recipes/configs/llama3_2/3B_lora_single_device.yaml>`_.
 
 This will fine-tune our model using a ``batch_size=2`` and ``dtype=bfloat16``. With these settings the model
```

Comment:
Batch size for this recipe is 4. Also I imagine the peak memory is no longer accurate?